pkg/distributors/contracts/MerkleOrchard.sol

```diff
@@ -125,6 +125,7 @@ contract MerkleOrchard {
                 if (currentChannelId != bytes32(0)) {
                     _setClaimedBits(liquidityProvider, currentChannelId, currentWordIndex, currentBits);
                     _deductClaimedBalance(currentChannelId, currentClaimAmount);
+                    emit DistributionSent(currentChannelId, recipient, tokens[claim.tokenIndex], currentClaimAmount);
```

Comment:
I've noticed that we're currently emitting `recipient` as `user` in the `DistributionSent` event. This seems odd to me as when using a callback contract the contract's address will be listed here instead of the `claimer`'s

Reply:
yes, recipient is a better term